ledger/acctonline_test.go

```diff
@@ -374,27 +376,19 @@ func TestAcctOnlineCache(t *testing.T) {
 	require.NoError(t, err)
 
 	for _, bal := range allAccts {
-		data, err := oa.accountsq.lookupOnline(bal.Addr, 0)
-		require.NoError(t, err)
-		require.Equal(t, bal.Addr, data.addr)
-		require.Equal(t, basics.Round(0), data.round)
-		require.Equal(t, bal.AccountData.MicroAlgos, data.accountData.MicroAlgos)
-		require.Equal(t, bal.AccountData.RewardsBase, data.accountData.RewardsBase)
-		require.Equal(t, bal.AccountData.VoteFirstValid, data.accountData.VoteFirstValid)
-		require.Equal(t, bal.AccountData.VoteLastValid, data.accountData.VoteLastValid)
-
+		// force populate the cache
 		oad, err := oa.lookupOnlineAccountData(0, bal.Addr)
 		require.NoError(t, err)
 		require.NotEmpty(t, oad)
 	}
 
 	// online accounts tracker requires maxDeltaLookback block to start persisting
-	targetRound := basics.Round(maxDeltaLookback * 2)
+	targetRound := basics.Round(maxDeltaLookback * numAccts * 2)
 	for i := basics.Round(1); i <= targetRound; i++ {
 		var updates ledgercore.AccountDeltas
 		acctIdx := (int(i) - 1) % numAccts
 
-		if int(i)%2 == 0 {
+		if (int(i)-1)%(numAccts*2) >= 5 {
```

Comment:
what's the idea behind this formula?

Reply:
I'm trying to do turn all the accts online, then all of them offline, then repeat.